enterprise/backend/src/metabase_enterprise/transforms/models/transform.clj

```diff
@@ -100,6 +100,23 @@
                                  tag-associations)]
       (for [transform transforms] (assoc transform :tag_ids (vec (get transform-id->tag-ids (:id transform) [])))))))
 
+(mi/define-batched-hydration-method transform-tag-names
```

Comment:
Note to self, opened https://linear.app/metabase/issue/DEV-1132/deprecateremove-define-batched-hydation-method-and-simple to track this